neuro_san/client/simple_one_shot.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
+from typing import Any
+from typing import Dict
+
+from neuro_san.client.agent_session_factory import AgentSessionFactory
+from neuro_san.client.streaming_input_processor import StreamingInputProcessor
+from neuro_san.interfaces.agent_session import AgentSession
+from neuro_san.message_processing.basic_message_processor import BasicMessageProcessor
+
+
+class SimpleOneShot:
+    """
+    Encapulates basic operation of sending a single message to an agent
+    and getting single aswer back.
+    """
```

Comment:
New client-side class to encapsulate sending a single message and getting a single answer.